weblate/templates/accounts/contact.html

```diff
@@ -12,9 +12,9 @@
 
 <div class="alert alert-info">
   <ul>
-    <li>{% blocktrans %}For questions about specific translation project, contact its maintainers directly, not by this form.{% endblocktrans %}</li>
-    <li>{% blocktrans %}You can only contact maintainers of the whole server here.{% endblocktrans %}</li>
-    <li>{% blocktrans %}If you seek support for Weblate, or want to file a bug-report, check its website at {{ weblate_link }}.{% endblocktrans %}</li>
+    <li>{% blocktrans %}Contact maintainers of this Weblate here.{% endblocktrans %}</li>
```

Comment:
```suggestion
    <li>{% blocktrans %}Contact maintainers of {{ site_title }} here.{% endblocktrans %}</li>
```

Reply:
The problem with this is if people don't change the site title...

Reply:
I am more worried they just call it "Weblate". Having it look wrong but understandable is sort of a corrective practice, seeing as the people in charge can be contacted right there and then. Maybe they would have had to also set up e-mailing for it to work, but alas.

TL;DR Made do without, since "itself" works nicely farther down, and the expected result always works. (Pending review.)